lib/polish_geeks/dev_tools/commands/git_conflicts.rb

```diff
@@ -0,0 +1,57 @@
+module PolishGeeks
+  module DevTools
+    module Commands
```

Comment:
@nijikon  wouldn't it be better to fgrep -R instead? It seams to me pretty heavy in terms of performance. Open each file, etc. What if there are huge binary files (images)? This version will use a looooooot of resources. I vote for fgrep -R